components/GoalItem.tsx

```diff
@@ -0,0 +1,24 @@
+import { StyleSheet, Text, View, Pressable } from 'react-native'
+import React from 'react'
+
+const GoalItem = (props) => {
+
+
+    return (
+        <View style={styles.goalItem}  >
+        <Pressable android_ripple={{color:"#dddddd"}} onPress={props.onDeleteItem.bind(this, props.id)}>
+            <Text style={{ color: "white",padding: 8, }}>{props.text}</Text></Pressable>
```

Comment:
use prettier in vs code to fix the indentation

Reply:
yes sir i will install